presto-client/src/client.ts

```diff
@@ -197,7 +226,7 @@ export class PrestoClient {
     const data = []
 
     do {
-      const response = await this.request({ method: 'GET', url: nextUri })
+      const response = await this.request({ headers, method: 'GET', url: nextUri })
```

Comment:
I'm sending ALL custom + X-Presto headers here on sub-sequent requests. Let me know if we should limit it. I haven't found any issues by doing so.
But as per @yhwang monitoring of `presto-cli`, it seems the CLI only sends X-Presto-User on these requests.